zeebe/exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/TestSupport.java

```diff
@@ -107,7 +107,8 @@ static Stream<ValueType> provideValueTypes() {
             ValueType.ROLE,
             ValueType.TENANT,
             ValueType.GROUP,
-            ValueType.MAPPING);
+            ValueType.MAPPING,
```

Comment:
❓ Can you remove MAPPING from here ? I just forgot to remove it when I implemented the exporters